network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -67,51 +70,165 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
         related_name='publication_hero_image',
         verbose_name='Publication Hero Image',
     )
+
```

Comment:
Using #8507 for reference, we have added the same customization fields to this page model.